example.env

```
@@ -0,0 +1 @@
+DB_HOST=mongodb+srv://abcd9076435:1789zxy@cluster0.wos6iur.mongodb.net/db-contacts?retryWrites=true&w=majority&appName=Cluster0
```

Całego nie umieszczaj, wystarczy tylko:
```suggestion
DB_HOST=
```